### PR TITLE
DC devices module

### PR DESCRIPTION
**Pull request by sfsyunusSLAC**
## Description
Device module for L2SI DC System controllers, ICT Power Distribution Unit and Meanwell Power Supply Unit

## Motivation and Context
Ophyd devices for these controllers, this module allows for automatic screen generation via Typhon for the PDUs and PSUs.

## How Has This Been Tested?
Test script in tests/test_dc_devices.py
This tests for controllable settings and readbacks

## Where Has This Been Documented?
Class docstrings contain documentation and usage instructions

### CODE REVIEW

**Comment by sfsyunusSLAC** [comment]
No support for Meanwell controllers at this moment, that will be added later

**Comment by ZLLentz** [comment]
Travis appears to be having a stroke. This PR is being associated with the tests for the previous PR that you closed, and the actual tests for this PR at https://travis-ci.org/pcdshub/pcdsdevices/builds/633967781?utm_medium=notification&utm_source=github_status are being marked as passing despite failing the unit test...

File: pcdsdevices/dc_devices.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    channel : ``str``
+        The output channel on the ICT, e.g. '1A', '3B', etc
+
+    Usage
+    ---------
+    ICTChannel('TST:PWR:ICT:01', '2B' )
+    """
+    Ch_Current = FCpt(EpicsSignalRO, '{self.prefix}:Output{self._ch}:Current',
+                      kind='hinted')
+    Ch_Status = FCpt(EpicsSignal, '{self.prefix}:Output{self._ch}:GetState',
```
Comment (ZLLentz) [inline pcdsdevices/dc_devices.py]: Your functions use `ch_status`, not `Ch_Status`. I suggest changing all the names to lowercase to match the python conventions.

Comment (sfsyunusSLAC) [reply pcdsdevices/dc_devices.py]: oops I did that while playing around with typhon, meant to set those back.

File: tests/test_dc_devices.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def fake_ICT():
+    FakeICT = make_fake_device(ICT)
+    PDU = FakeICT('TST:PDU:ICT', name='test')
+    PDU.ch1A.ch_status.sim_set_enum_strs(['ENABLED', 'DISABLED'])
+    PDU.ch1A.ch_status.sim_put('ENABLED')
```
Comment (ZLLentz) [inline tests/test_dc_devices.py]: The automated tests are failing at setup because in the class you use `ch_1A` but in the tests you use `ch1A`.

**Pull request merged**